apps/api/src/guards/has-permissions.guard.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  Injectable,
+  CanActivate,
+  ExecutionContext,
+  HttpException
+} from '@nestjs/common';
+import { Reflector } from '@nestjs/core';
+import { StatusCodes, getReasonPhrase } from 'http-status-codes';
+import { HAS_PERMISSION_KEY } from '@ghostfolio/api/decorators/has-permission.decorator';
+import { hasPermission } from '@ghostfolio/common/permissions';
+
+@Injectable()
+export class HasPermissionsGuard implements CanActivate {
```

Comment:
Can you rename to `HasPermissionGuard` because it checks only for one permission, right?

Reply:
done